packages/ui/src/components/MaterialsCard/MaterialsCard.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { MaterialsCard } from './MaterialsCard';
+
+describe('MaterialsCard', () => {
+  it('should renders correctly in idle status', () => {
+    render(<MaterialsCard status="idle" />);
+
+    const button = screen.getByText('Wygeneruj listę');
```

Comment:
🤔  Shouldn't we import those texts? We've already defined them in the component file